### ISSUE TITLE
Is it possible to calculaealculating quantities for trackers that depend on their host halo properties?

### ISSUE THREAD

**Issue by Martin-Rey** [pynbody/tangos#195]
Hi,

I have a question on the best-use of trackers and whether it is possible to compute properties for them that depends on other halo properties.

For example, I have identified a stellar sub-population with specific kinematics at z=0 that I am interested in. I then defined a `tracker` object to identify it over the galaxy's history, and would like to compute its phase space over time (velocities, radii, orbital energy etc).

This works very well for mass, but a lot of quantities like velocities are frame-dependent and most meaningful when defined against the entire stellar population of the galaxy. That means accessing e.g. the `shrink_center` and `faceon` orientation of the total host halo, rather than calculating it from the particles solely available inside the tracker.

Is there a way to natively link trackers to a parent halo (maybe using the `link` infrastructure) , and to access the host properties to define any subsequent trackers' properties?

Martin

**Comment by apontzen** [comment]
I don’t believe this is currently supported, but the way I’d recommend doing it is to extend the idea of crosslinking to be able to identify tracker host halos through the particles in common. It might be worth checking how black holes find their `host_halo` link, though I suspect it’s simpler in that case as it’s only one particle.
